ils_middleware/dags/stanford.py

```diff
@@ -71,6 +71,9 @@
         convert_to_symphony_json = PythonOperator(
             task_id="convert_to_symphony_json",
             python_callable=to_symphony_json,
+            op_kwargs={
+                "marc_json": "{{ task_instance.xcom_pull(task_ids='marc_json_to_s3', key='marc_json') }}"
```

Comment:
wouldn't the `marc_json_to_s3` task push an xcom record to the `return_value` key, with a `marc_json` field in the hash stored in the xcom record?  i don't see anything in `send_to_s3` that does an `xcom_push` to `marc_json`.

Reply:
Thanks! Call is now fixed to use the return value. Returned JSON is serialized as a string and not a dictionary with `marc_json`.